templates/pages/checkout.html

```diff
@@ -17,7 +17,7 @@ <h1 class="is-srOnly">{{lang 'checkout.title'}}</h1>
         <h2 class="checkoutHeader-heading">
             <a class="checkoutHeader-link" href="{{urls.home}}">
                 {{#if checkout.header_image}}
-                    <img alt="{{settings.store_logo.title}}" class="checkoutHeader-logo lazyload" id="logoImage" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{ checkout.header_image }}"/>
+                    <img alt="{{settings.store_logo.title}}" class="checkoutHeader-logo" id="logoImage" data-sizes="auto" src="{{ checkout.header_image }}"/>
```

Comment:
Is `data-sizes` only required for lazy loading?

Reply:
I believe so. @BC-EChristensen can you confirm the above.